app/routes/v2/clusters.js

```diff
@@ -122,13 +122,25 @@ var runAddClusterWebhook = async(req, orgId, clusterId, clusterName)=>{
   }
 };
 
-function pushToS3Sync(key, searchableDataHash, dataStr, data_location, logger) {
+function pushToS3Sync(org, key, searchableDataHash, dataStr, data_location, logger) {
   //if its a new or changed resource, write the data out to an S3 object
   const result = {};
-  const bucket = conf.storage.getResourceBucket(data_location);
+  // const bucket = conf.storage.getResourceBucket(data_location);
+  const locationConf = conf.storage.s3ConnectionMap.get(data_location || conf.storage.defaultLocation) || {};
+  const bucketPrefix = locationConf.orgBucketPrefix || 'razee-org-';
+  const bucket = `${bucketPrefix}${org._id.toLowerCase()}`;
```

Comment:
Why toLowerCase?  I wouldn't think uppercase letters would be a problem for bucket names, and it unnecessarily obscures the 'true' org ID.  I'm not aware of any code dependent on parsing the 'true' org ID with correct capitalization from the name (the COS backup code no longer does for example), but this could introduce unnecessary confusion for SREs.
```suggestion
  const bucket = `${bucketPrefix}${org._id}`;
```